cve_bin_tool/NVDAutoUpdate.py

```diff
@@ -177,13 +133,10 @@ def extract_data(nvddir):
         except Exception as exception:
             print("Exception in extract_data: " + str(exception))
 
-
-    """ Extract curl data using script """
-    find_curl_list(cve_number, vendor_name, product_name, exploitability_score, impact_score, severity, versions)
```

Comment:
Instead of calling many functions individually from every source , we can call only once function.

Reply:
Don't know if we need this PR. But I feel that `find_curl_list` should not be in `NVDAutoUpdate.py`.